app/controllers/api/v1/inventory_columns_controller.rb

```diff
@@ -6,7 +6,7 @@ class InventoryColumnsController < BaseController
       before_action :load_team
       before_action :load_inventory
       before_action :load_inventory_column, only: %i(show update destroy)
-      before_action :check_manage_permissions, only: %i(create update destroy)
```

Comment:
You removed this check, but didn't add anything to create method?

Reply:
yeah, i thought all users should be able to create columns on inventory, my mistake

Reply:
will attempt to fix all of this tommorow